src/main.rs

```diff
@@ -1,20 +1,26 @@
+#[allow(dead_code)] // to reduce warning spam
```

Comment:
Would rather actually fix the dead code then suppress warnings

Reply:
There is alot of dead code that will be used later like the player(s) so I thought hiding them can help